### PR TITLE
Added preview function

### PR DESCRIPTION
**Pull request by kricli**

### CODE REVIEW

File: js/application.js [hunk 1]
```diff
@@ -345,11 +345,11 @@ $(document).ready(function() {
         $('.stopWatch').text(time);
       })
       //Speed choosing
-      $('#verySlow').click(function() {$('#buttonGroupOriginal button').removeClass('active'); $(this).addClass('active'); interval = 400})
-      $('#slow').click(function() {$('#buttonGroupOriginal button').removeClass('active'); $(this).addClass('active'); interval = 250})
-      $('#normal').click(function() {$('#buttonGroupOriginal button').removeClass('active'); $(this).addClass('active'); interval = 150})
-      $('#fast').click(function() {$('#buttonGroupOriginal button').removeClass('active'); $(this).addClass('active'); interval = 100})
-      $('#veryFast').click(function() {$('#buttonGroupOriginal button').removeClass('active'); $(this).addClass('active'); interval = 50})
+      $('#verySlow').click(function() { $('#buttonGroupOriginal button').removeClass('active'); $(this).addClass('active'); interval = 400})
```
Comment (h8rry) [inline js/application.js:348]: Some of these codes look like they can be refactored.

For example, you used 5 IDs here, but you can use 1 Class + 5 different data attributes. This way, it makes your code more modular.

For example

``` js

$('.speed').click(function() {
  $('#buttonGroupOriginal button').removeClass('active');
  $(this).addClass('active'); 

  category = $('.speed').attr('data-category');

  switch(category) {
    case "verySlow":
      code block
      break;
    case "slow":
      interval = 250;
      break;
  }
});
```

Comment (h8rry) [inline js/application.js:348]: Try to go through your code and improve / rewrite them if needed / possible!

**Comment by h8rry** [comment]
The rest looks fine

File: js/application.js (hunk 1 above)
Comment (kricli) [inline js/application.js:348]: I am just going to keep this for now. Will improve it when I have time.

Comment (h8rry) [inline js/application.js:348]: Ok

**Pull request merged**